parcels/fieldset.py

```diff
@@ -109,6 +108,8 @@ def check_complete(self):
                 if g.defer_load:
                     g.time_full = g.time_full + (g.time_origin - ugrid.time_origin) / np.timedelta64(1, 's')
                 g.time_origin = ugrid.time_origin
+        if getattr(self, 'UV') is None:
+            self.add_field(VectorField('UV', self.U, self.V))
```

Comment:
It's somewhat confusing to add a `VectorField` with a `add_field` method. Should we create a new method `add_vectorfield`? Especially because half of the lines of `add_field` only apply to `Fields`